Sources/Build/Sanitizers.swift

```diff
@@ -39,17 +45,18 @@ public struct EnabledSanitizers {
 
     /// Sanitization flags for the C family compiler (C/C++).
     public func compileCFlags() -> [String] {
-        return sanitizers.map({ "-fsanitize=\($0.rawValue)" })
+        return sanitizers.map{ $0.flagName }
     }
 
     /// Sanitization flags for the Swift compiler.
     public func compileSwiftFlags() -> [String] {
-        return sanitizers.map({ "-sanitize=\($0.rawValue)" })
+        // UBSan is only available for C family languages
+        return sanitizers.filter{ $0 != .undefined }.map{ $0.flagName }
```

Comment:
this used to return `-sanitize=asan` and now is `-fsanitize=asan`, that's intentional?

Reply:
Wait, this is an old commit, one sec

Reply:
@keith - thats actually not correct though, before it was `-sanitize=address` and the `asan`/`tsan` values were used to create the dylib path to the santizer runtime